template/cordova/lib/build.js

```diff
@@ -411,92 +437,6 @@ function clearIntermediatesAndGetPackage (bundleTerms, config, hasAnyCpu) {
     return pckage.getPackageFileInfo(finalFile);
 }
 
-// Can update buildConfig in the following ways:
-//  * Sets targetProject property, the project to launch when complete
-function getBuildTargets (buildConfig) {
```

Comment:
I think I chose this one to remove because the other one was closer to what will be kept in use. 
With the export I technically agree, but this is quite a mess anyway and should probably be cleaned up in a future PR.